google-cloud-vision/samples/async_batch_annotate_images.rb

```diff
@@ -0,0 +1,76 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# sample-metadata
+#   title: Async Batch Image Annotation
+#   description: Perform async batch image annotation
+
+require "google/cloud/vision"
```

Comment:
Should these requires be inside the region tags (usually also inside the sample method?)

Reply:
Oh. Yeah good catch, I think so. I copied these verbatim from the original orphaned branch on ruby-docs-samples and didn't pay enough attention. Fixed.